build.sh

```diff
@@ -1,6 +1,3 @@
 #!/bin/bash
-mkdir -p bin
-cd bin
-cmake -S ../ -B .
-make && make Shaders && ./vged
-cd ..
+cmake -S . -B bin
```

Comment:
change it to build instead of bin because bin means executable. We are are going to have a lot of them

Reply:
ok